sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -299,17 +298,21 @@ def test_split_points(self):
     for _ in splits[0].source.read(range_tracker):
       split_points_report.append(range_tracker.split_points())
 
-    # There are a total of three blocks. Each block has more than 10 records.
+    # There will be a total of num_blocks in the generated test file,
+    # proportional to number of records in the file divided by syncronization
+    # interval used by avro during write. Each block has more than 10 records.
+    num_blocks = int(math.ceil(14.5 * num_records /
+                               avro.datafile.SYNC_INTERVAL))
 
     # When reading records of the first block, range_tracker.split_points()
     # should return (0, iobase.RangeTracker.SPLIT_POINTS_UNKNOWN)
-    self.assertEquals(
+    self.assertEqual(
```

Comment:
Please add an assertion for num_blocks > 1 here.

Reply:
Done, thanks you!